GitTabExpansion.ps1

```diff
@@ -74,6 +74,12 @@ function script:gitBranches($filter, $includeHEAD = $false) {
         foreach { $prefix + $_ }
 }
 
+function script:gitTags($filter) {
+    git --git-dir="$(Get-GitDirectory)" for-each-ref --format='%(refname:short)' refs/tags |

```

Comment:
As we celebrate the anniversary of this comment... do you want to take a crack at updating this or should I?